### PR TITLE
Fixed copyright for SDK

### PR DESCRIPTION
**Pull request by marcalff**
Fixed copyright for every file under sdk.

Removed unused `sdk/CHANGELOG.md`

File `sdk/src/trace/samplers/trace_id_ratio.cc` is unchanged,
to discuss separately (different license)

## Changes

Fixed copyright.

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

File: buildscripts/pre-commit [hunk 1]
```diff
@@ -48,6 +48,9 @@ git add "$(pwd)/api/include/opentelemetry/version.h"
 
 # Update sdk version.cc
 cat > "$(pwd)/sdk/src/version/version.cc" <<END
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
```
Comment (lalitb) [inline buildscripts/pre-commit:53]: This can be removed :)

**Review by ThomsonTan — APPROVED** [review]
Thanks for fixing this.

**Review by lalitb — APPROVED** [review]
LGTM.

File: buildscripts/pre-commit (hunk 1 above)
Comment (lalitb) [reply buildscripts/pre-commit:53]: Oh please ignore. Looks good .

**Pull request merged**